edit/tty/reader.go

```diff
@@ -148,7 +148,14 @@ func (rd *Reader) readOne(r rune) {
 
 	switch r {
 	case 0x1b: // ^[ Escape
+		var prefixMod ui.Mod
 		r2 := readRune()
+		// some terminals may also send Alt character which is same as ESC
```

Comment:
Elvish already handles the use of Alt-sends-Escape in some cases (near the end of this function, search "Something other than"), and this branch is specifically for handling double ESCs. I would rather you clarify this comment to make it read like:

> According to https://unix.stackexchange.com/a/73697, rxvt and derivatives prepend another ESC to a CSI-style or G3-style sequence to signal Alt. If that happens, remember this now; it will be later picked up when parsing those two kinds of sequences.

Also, I'd rather you make this a boolean flag `hasTwoLeadingESC`; a Mod seems needlessly general.

Reply:
Done.